apps/native-component-list/src/screens/Audio/expo-audio/RecordingScreen.tsx

```diff
@@ -15,7 +15,7 @@ export default function RecordingScreen() {
     return uri ? (
       <>
         <HeadingText>Last recording</HeadingText>
-        <AudioPlayer source={{ uri }} />
+        <AudioPlayer key={uri} source={{ uri }} />
```

Comment:
The player would often not update in the Emulator when I made quick recordings back to back, but adding a key fixed that issue.